### DISCUSSION TITLE
The Long Dark

### DISCUSSION THREAD

**Discussion by TheMercenary333** [skyline-emu/skyline#1475] [Game Requests]
![egs-thelongdark-hinterlandstudioinc-g1a-00-1920x1080-231d3328f957](https://user-images.githubusercontent.com/74915480/172027402-ea2a2116-024c-456c-a0f4-191e80301178.jpg)

**Comment by raviii12** [comment]
Is it boot

**Reply by Crytonics** [reply]
> Is it boot

If you can test it, you can post it in game list
